example-apps/kitchen-sink-new-router/source/components/angular-directives/hello.ts

```diff
@@ -0,0 +1,12 @@
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'hello',
+  inputs: ['msg'],
```

Comment:
@sumitarora @ericjim I think that the preferred way is to use `@Input` decorator instead no?